protocol/x/clob/keeper/orders.go

```diff
@@ -521,6 +522,12 @@ func (k Keeper) PlaceStatefulOrdersFromLastBlock(
 		}
 
 		order := orderPlacement.GetOrder()
+
+		// Skip the order if it is a post-only order and we are only placing post-only orders.
```

Comment:
Nit: comment could be more accurate/complete - right now it just describes one of the two scenarios